### PR TITLE
feat(pool): pool committed size

### PR DESCRIPTION
**Pull request by dsavitskiy**

### CODE REVIEW

**Comment by dsavitskiy** [comment]
bors try

**Review by tiagolobocastro — APPROVED** [review]

**Review by Abhinandan-Purkait — APPROVED** [review]

File: io-engine-tests/src/pool.rs [hunk 1]
```diff
@@ -91,6 +92,20 @@ impl PoolBuilder {
                 Status::new(Code::NotFound, format!("Pool '{uuid}' not found"))
             })
     }
+
+    pub async fn get_replicas(&self) -> Result<Vec<Replica>, Status> {
```
Comment (tiagolobocastro) [inline io-engine-tests/src/pool.rs:96]: small nit: we have list_pools but get_replicas, was this intended?

**Comment by dsavitskiy** [comment]
bors merge

**Comment by tiagolobocastro** [comment]
bors merge

**Comment by tiagolobocastro** [comment]
bors merge

**Pull request merged**